content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1033,6 +1033,24 @@ and legacy in-tree cloud providers), it sets this annotation on the Node to deno
 set from the command line flag (`--node-ip`). This IP is verified with the cloud provider as valid
 by the cloud-controller-manager.
 
+### volume.alpha.kubernetes.io/node-affinity (alpha) {#volume-alpha-kubernetes-io-node-affinity}
+
+Type: Annotation
+
+Example: `volume.alpha.kubernetes.io/node-affinity: "constrain-with-nodes"`
+
+Used on: Node
+
+The `volume.alpha.kubernetes.io/node-affinity` annotation is used to specify node affinity rules for a 
+PersistentVolume (PV) in Kubernetes. It allows you to restrict the scheduling of Pods that use the corresponding 
+PersistentVolume to specific nodes in the cluster, based on [node labels](/docs/concepts/storage/persistent-volumes/#selector).
+
+Node affinity and [nodeSelector](https://github.com/kubernetes/website/blob/main/content/en/docs/reference/labels-annotations-taints/_index.md#scheduleralphakubernetesionode-selector-schedulerkubernetesnode-selector) are conceptually similar, both enabling you to restrict the scheduling 
```

Comment:
Make sure that you're wrapping lines at 80 characters for ease of reviewing (unless a link URL exceeds the 80 chars which is fine)